shared/src/business/useCases/externalDocument/generateExternalDocumentTitle.ts

```diff
@@ -10,7 +11,7 @@ import { cloneDeep } from 'lodash';
  * @returns {string|void} document title
  */
 export const generateExternalDocumentTitle = (
-  applicationContext: IApplicationContext,
+  applicationContext: ClientApplicationContext,
```

Comment:
Should this move to `web-client/` if it is going to be typed this way?

Reply:
Done!